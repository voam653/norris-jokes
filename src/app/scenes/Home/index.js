import React, { Component } from 'react';
import { Container } from './style';
import JokeModal from '../../components/JokeModal';
import Header from '../../components/Header';

class Home extends Component {
    render() {
        return (
            <Container>
                <Header />
                <ul>
                    <li>
                        <div>
                            <button type="button" data-toggle="modal" data-target="#jokeModal">
                                Animals
                            </button>
                        </div>
                    </li>
                </ul>
                <JokeModal />
            </Container>
        )
    }
}

export default Home;