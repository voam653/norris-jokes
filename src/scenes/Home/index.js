import React, { Component } from 'react';
import { Container } from './style';

class Home extends Component {
    render() {
        return (
            <Container>
                Home
            </Container>
        )
    }
}

export default Home;