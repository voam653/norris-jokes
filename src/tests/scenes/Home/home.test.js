/**
 * @jest-environment jsdom
 */
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Home from '../../../app/scenes/Home';

describe('HomeComponent', () => {
    const renderer = new ShallowRenderer();

    describe('Snapshots', () => {
      it('Should be a simple HomeComponent', () => {
        const component = renderer.render(
          <Home />,
        );
        expect(component).toMatchSnapshot();
      });
    });
});