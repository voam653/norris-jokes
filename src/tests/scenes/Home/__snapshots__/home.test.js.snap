// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeComponent Snapshots Should be a simple HomeComponent 1`] = `
<ForwardRef(styled.div)>
  <Header />
  <UNDEFINED />
  <UNDEFINED />
</ForwardRef(styled.div)>
`;
